cmd/ekanited/main.go

```diff
@@ -70,8 +75,9 @@ func main() {
 		batchSize       = fs.Int("batchsize", DefaultBatchSize, "Indexing batch size")
 		batchTimeout    = fs.Int("batchtime", DefaultBatchTimeout, "Indexing batch timeout, in milliseconds")
 		indexMaxPending = fs.Int("maxpending", DefaultIndexMaxPending, "Maximum pending index events")
-		tcpIface        = fs.String("tcp", DefaultTCPServer, "Syslog server TCP bind address in the form host:port. To disable set to empty string")
-		udpIface        = fs.String("udp", "", "Syslog server UDP bind address in the form host:port. If not set, not started")
+		tcpIface        = fs.String("tcp", DefaultTCPServer, "Server TCP bind address in the form host:port. To disable set to empty string")
+		udpIface        = fs.String("udp", "", "Server UDP bind address in the form host:port. If not set, not started")
+		inputType       = fs.String("input", DefaultInputType, "Input type format. Chose between syslog and json.")
```

Comment:
I think in this case it makes more sense to keep the abbreviation *JSON* in lowercase letters, because no other flag value contains an uppercase letter,  so it's more consistent.